roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -9,8 +9,10 @@ openshift_logging_mux_namespace: logging
 
 ### Common settings
 openshift_logging_mux_nodeselector: "{{ openshift_hosted_logging_mux_nodeselector_label | default('') | map_from_pairs }}"
-openshift_logging_mux_cpu_limit: 500m
-openshift_logging_mux_memory_limit: 1Gi
+openshift_logging_mux_cpu_limit: "{{ openshift_hosted_logging_mux_cpu_limit | default('500m') }}"
```

Comment:
I don't think we should add them if they don't currently exist.
With Openshift-Ansible 3.4 and prior, the variables to configure openshift-logging were named `openshift_hosted_logging_*` to match the role that installed them.

Since we those variables didn't exist for Mux, I think we can just omit using them with defaults.